amd_openvx/openvx/hipvx/vision_kernels.cpp

```diff
@@ -249,6 +252,8 @@ int HipExec_CannySobel_U16_U8_3x3_L1NORM(hipStream_t stream, vx_uint32 dstWidth,
                         dim3(localThreads_x, localThreads_y), 0, stream, dstWidth, dstHeight, (uchar *)pHipDstImage , dstImageStrideInBytes,
                         (const uchar *)pHipSrcImage, srcImageStrideInBytes);
 
+    hipDeviceSynchronize();
```

Comment:
why do you need this here
remove print

Reply:
this is for testing purposes. I'll let you know when the PR is ready for review